tests/testthat/test-enw_quantiles_to_long.R

```diff
@@ -7,4 +7,15 @@ test_that("enw_quantiles_to_long can manipulate posterior draws as expected", {
         c("rhat", "ess_bulk", "ess_tail") := NULL][!is.na(variable)][]
     )
   )
-})
\ No newline at end of file
+})
+
+test_that("enw_quantiles_to_long can receive data.frame", {
+  # When a data.table is passed to `melt()`, it calls `data.table::melt`, but if
+  # anything else is passed to `melt()`, it calls `reshape2::melt`, which is
+  # deprecated
+  # See this StackOverflow answer
+  # https://stackoverflow.com/questions/42141989/why-do-i-get-this-error-message-could-not-find-function-patterns-when-using-t
+  fit <- enw_example("nowcast")
+  smry <- as.data.frame(summary(fit))
+  expect_no_error(enw_quantiles_to_long(smry))
+})
```

Comment:
does this need a new line?

Reply:
There is a new line, but it doesn't show in the diff. If you click the three dots in the corner of the diff for this file, and then click "Edit file", you can see the new line.